### DISCUSSION TITLE
Ability to customize commit message

### DISCUSSION THREAD

**Discussion by cyberhck** [BetaHuhn/repo-file-sync-action#87] [Q&A] [answered]
We use conventional commit message format, and the format which this action provides isn't a valid convention.

So we should either: change the commit message moving forward to use that format, OR a config which lets me customize the commit message format.

I'd recommend going for first option because:
- the people who don't use that format most likely don't care if you use that format.
- the people who do use that format, will be happy with it.
- a lot of automated tools already use that format by default (dependabot, snyk, greenkeeper when it was active, etc)

**Comment by cyberhck** [comment, marked as answer]
actually, looking at all config, while not fully customizeable, I can add a prefix to message, so I think that's kinda enough, it'd be nice to support fully customizable things, but it's fine I guess :)

**Reply by BetaHuhn** [reply]
Yes, you can already customize quite a few things like the `PR_BODY`, `COMMIT_BODY` and `COMMIT_PREFIX`. Also take a look at https://github.com/BetaHuhn/repo-file-sync-action/discussions/13 and let me know if that would be helpful to you.

**Reply by cyberhck** [reply]
actually the #13 would be awesome, it'd be more awesome than what I was requesting here :slightly_smiling_face: as soon as that is done, I'll use that.

Thanks, I didn't know that was in the works.

**Reply by BetaHuhn** [reply]
Great, marking this as solved.
